### PR TITLE
[MoM] Add the Megakinesis telekinetic power

### PR DESCRIPTION
**Pull request by Standing-Storm**
#### Summary
Mods "[MoM] Add the Megakinesis telekinetic power"

#### Purpose of change

Following up on Force Shove being affected by weight, add Megakinesis.

#### Describe the solution

Add the Megakinesis power, which functions like Force Shove but with a much higher weight limit (150 kg * power level + 100, vs. 25 * power level + 10). Even a basic megakinesis usage can splatter a zombie all over the walls, and it's much more possible to hurl hulks with it.

However, Megakinesis costs much more (9000 starting vs 1750 starting) and at Difficulty 8, has a much greater chance of causing Nether Attunement.

#### Testing

Power works, contemplating it works, pushing and pulling both work. 

#### Additional context

At first I was like "Is it okay to have a higher level power that does the same conceptual thing as a lower level power?" but I already did that with Fountain of Flames / Crucible (single-target fire damage, former is Difficulty 2 and latter is Difficulty 10). And anyway, it allows for tactical decisions (can I get by with the lower-cost power that's less risky to use, or do I need to break out the big guns?)

### CODE REVIEW

**Comment by Hyperseeker** [comment]
Have you considered side upgrades for powers? Instead of implementing a new subfeature of a spell with a separate power, you could add a trait that allows... I dunno, taking on a much-heavier weight with Force Hand? Maybe it changes the weight factor so that weight becomes less of a detriment when using Force Hand?

**Comment by Standing-Storm** [comment]
Due to the way the systems for Nether Attunement and so on work, that would be totally untenable unless I only do it once or twice (since I'd need to write a special exception for every single power that broke from the current formula of "the chance of Nether Attunement is the power Difficulty squared, the chance for Nether Attunement effects to trigger is affected by the Difficulty, etc", and the high cost/Nether Attunement chance is an impoortant balancing factor, otherwise it's just back to "Get the trait and you can Force Shove shoggoths a couple times to kill then" like beffore ). There's no good infrastructure for tying powers together in any real way.

File: data/mods/MindOverMatter/recipes/practice/telekinesis_practice.json [hunk 1]
```diff
@@ -993,6 +994,83 @@
...
+    "proficiencies": [ { "proficiency": "prof_contemplation_telekinesis", "required": false } ],
+    "tools": [ [ [ "matrix_crystal_drained", -1 ], [ "matrix_crystal_telekinesis", -1 ] ] ],
+    "components": [ [ [ "matrix_crystal_telekin_dust", 1 ] ] ],
+    "flags": [ "SECRET", "BLIND_HARD" ],
+    "result_eocs": [
+      {
+        "id": "EOC_PRACTICE_MOVE_LARGE_WEIGHT",
+        "condition": { "math": [ "u_spell_level('telekinetic_move_large_weight')", ">=", "1" ] },
+        "effect": [
+          { "u_message": "You spend some time meditating and contemplating your powers and emerge with new knowledge." },
```
Comment (casswedson) [inline data/mods/MindOverMatter/recipes/practice/telekinesis_practice.json:1018]:
```suggestion
          {
            "u_message": "You spend some time meditating and contemplating your powers and emerge with new knowledge.",
            "type": "good"
          },
```

Comment (Standing-Storm) [reply data/mods/MindOverMatter/recipes/practice/telekinesis_practice.json:1018]: Oh, right 😅

**Pull request merged**